src/ebddocx2table/docxtableconverter.py

```diff
@@ -263,6 +266,74 @@ def _handle_single_table(
                     )
                 )
 
+    # see above boolean_outcome and subsequent_step_number could be ignored iff schemes of *-numbers are always the same
+    # pylint:disable=too-many-locals
+    def _handle_single_table_star_exception(
+        self,
+        table: Table,
+        multi_step_instructions: List[MultiStepInstruction],
+        row_offset: int,
+        rows: List[EbdTableRow],
```

Comment:
```suggestion
        rows: list[EbdTableRow],
```
https://peps.python.org/pep-0585/

Reply:
5389706b70210cda5683ea96af2abe6fa46cb473